app/assets/javascripts/application.js

```diff
@@ -6,6 +6,8 @@
 //
 //= require govuk_publishing_components/lib
 //= require govuk_publishing_components/components/button
```

Comment:
Looks like we're removing button from the CSS but leaving it in the JS. Button is used in this application but although the button CSS is included in static it doesn't look like the button JS is - which means this current configuration works, but is arguably incorrect - we should include button JS in static and remove it from here.

Reply:
I've raised a pull request to add it to `static`: https://github.com/alphagov/static/pull/2820